src/public/locales/fr/default_translation_fr.json

```diff
@@ -77,5 +76,15 @@
   "or": "ou",
   "Ingestion History Is Empty": "L'historique d'ingestion est vide",
   "Latest Data Ingestion": "Dernière ingestion de données",
+  "Beacon error": "Erreur de Beacon",
+  "variants form should include either an end position or both reference and alternate bases": "Le formulaire de variants doit inclure soit la position de fin, ou la référence et les bases alternatives",
```

Comment:
Le formulaire de recherche de variants doit inclure soit la position de fin, ou la base de référence et la base alternative.

Reply:
i think beacon can search not just SNVs, so plural bases would be correct?

Reply:
also, the inconsistent capitalization here might be rendered oddly, maybe this should be checked